build fails on OS X

I'm trying to build the test app on OS X and I'm getting some errors.
When I run script/build
I get this:

```
Building without unit tests...
-- The C compiler identification is AppleClang 6.0.0.6000056
-- The CXX compiler identification is AppleClang 6.0.0.6000056
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Found PkgConfig: /usr/local/bin/pkg-config (found version "0.28") 
-- PkgConfig was found.
-- checking for module 'libusb-1.0'
--   found libusb-1.0, version 1.0.9
-- libusb was found: usb-1.0
-- Configuring done
-- Generating done
-- Build files have been written to: /Volumes/HDD/Downloads/libblinkstick-master/build
Scanning dependencies of target libblinkstick
[ 50%] Building C object src/CMakeFiles/libblinkstick.dir/libblinkstick.c.o
Linking C static library ../../target/liblibblinkstick.a
[ 50%] Built target libblinkstick
Scanning dependencies of target blinkstick
[100%] Building C object src/CMakeFiles/blinkstick.dir/blinkstick.c.o
Linking C executable ../../target/blinkstick
ld: library not found for -lusb-1.0
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [../target/blinkstick] Error 1
make[1]: *** [src/CMakeFiles/blinkstick.dir/all] Error 2
make: *** [all] Error 2
```

How could I fix this?

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -12,13 +12,6 @@ else(PKG_CONFIG_FOUND)
   message(FATAL_ERROR "Could not find PkgConfig")
 endif(PKG_CONFIG_FOUND)
 
-# Have we found libusb?
-if(LIBUSB_FOUND)
-  message(STATUS "libusb was found: ${LIBUSB_LIBRARIES}")
-else(LIBUSB_FOUND)
-  message(FATAL_ERROR "Could not find libusb.")
-endif(LIBUSB_FOUND)
-
 option(test "Build all tests." OFF)
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -m64 -std=c++0x -stdlib=libc++ -g3 -Wall -O0  -arch x86_64")
 SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -m64 -arch x86_64")

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -12,6 +12,8 @@ else(PKG_CONFIG_FOUND)
   message(FATAL_ERROR "Could not find PkgConfig")
 endif(PKG_CONFIG_FOUND)
 
+link_directories(${LIBUSB_LIBRARY_DIRS})
+
 option(test "Build all tests." OFF)
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -m64 -std=c++0x -stdlib=libc++ -g3 -Wall -O0  -arch x86_64")
 SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -m64 -arch x86_64")

